### PR TITLE
Headbars: Add hp_max

### PR DESCRIPTION
**Pull request by ghost**
headbars.attach_hpbar(object) -> headbars.attach_hpbar(object, max_hp)

### CODE REVIEW

File: mods/headbars/init.lua [hunk 1]
```diff
@@ -46,39 +44,31 @@ minetest.register_entity(
       collisionbox = {0, 0, 0, 0, 0, 0},
 
       on_step = function(self, dtime)
-		   local ent = self.wielder
+		   local player = self.wielder and inetest.get_player_by_name(self.wielder)
```
Comment (SmallJoker) [inline mods/headbars/init.lua]: `inetest`

**Comment by kaadmy** [comment]
I didn't want to use anything player-specific, because I was also thinking of attaching it to mobs/bosses too.

**Comment by Rui914** [comment]
Update.
I changed the contents.

**Comment by ghost** [comment]
Update.

**Pull request closed without merging**